.github/pull_request_template.md

```diff
@@ -1,29 +1,28 @@
-<!-- Replace this line with what this PR does and why.  Describe what you'd like reviewers to know, how you applied the Engineering principles, and any interesting tradeoffs made.  Delete bullet points below that don't apply, and update the changelog section as appropriate. -->
+<!-- Replace this block with what this PR does and why. Describe what you'd like reviewers to know, how you applied the engineering principles, and any interesting tradeoffs made. Delete bullet points below that don't apply, and update the changelog section as appropriate. -->
 
 <!-- Add JIRA link here. Short version (e.g. MTT-123) also works and gets auto-linked. -->
 
 <!-- Add RFC link here if applicable. -->
 
 ## Changelog
 
-### com.unity.netcode.gameobjects
```

Comment:
removed section separation since there is no other package under this repo anymore